apps/range-voting/app/components/Slider.js

```diff
@@ -52,7 +52,7 @@ class Slider extends React.Component {
   updateValueFromClientX(clientX) {
     const rect = this.getRect()
     const x = parseFloat(
-      Math.min(rect.width, Math.max(0, clientX - rect.x)).toFixed(2)
+      Math.min(rect.width, Math.max(0, clientX - rect.x))
```

Comment:
If you are not using toFixed then the value will not be converted to number, so it does not need to use parseFloat anymore